test/client/client.go

```diff
@@ -19,23 +19,25 @@ package main
 import (
 	"context"
 	"flag"
+	"fmt"
 	"io/ioutil"
 	"log"
 	"net/http"
+	"os"
 	"strings"
 	"time"
 
+	"google.golang.org/grpc"
+
 	"crypto/tls"
 
 	"github.com/golang/sync/errgroup"
+	pb "istio.io/manager/test/grpcecho"
```

Comment:
copy paste error from grpc-go tutorial... I don't mind keeping the original name.